docs/en/scanners.md

```diff
@@ -11,7 +11,11 @@ need to swap it out with an alternative scanner.
 ## Keypad Scanners
 The scanners in `kmk.scanners.keypad` wrap the `keypad` module that ships with
 CircuitPython and support the same configuration and tuning options as their
-upstream. You can find out more in the [CircuitPython
+upstream.
+
+**Some users may need to tweak debounce parameters**, `interval=0.01,debounce_threshold=5` is a good starting point. `debounce_threshold` is only applicable for CircuitPython >= 9.2.0
```

Comment:
```suggestion
**Some users may need to tweak debounce parameters**, `interval=0.01,debounce_threshold=5` is a good starting point. `debounce_threshold` is only applicable for CircuitPython >= 9.0.0
```

Pretty sure that's CP9.0.0 . I remember porting from 8 to 9 before 9 was out and daily driving a janky alpha build.

Reply:
Well, it didn't work on 9.0.5